src/openforms/emails/tasks.py

```diff
@@ -5,13 +5,11 @@
 from django.utils import timezone
 from django.utils.translation import gettext_lazy as _
 
-from django_yubin.models import Message
-
 from openforms.celery import app
 from openforms.config.models import GlobalConfiguration
-from openforms.logging.models import TimelineLogProxy
+from openforms.registrations.utils import collect_registrations_failures
```

Comment:
there's a remark further down suggesting to move this to `openforms.logging.service`